### PR TITLE
Adjustment/change userlogin page UI

### PR DESCRIPTION
**Pull request by maxellmilay**
- changed login input data retrieval from onSubmit form to onClick button
 - setup Material UI for the project
 - implemented ui changes using Material UI
 - created styles folder

### CODE REVIEW

File: client/src/pages/UserLogin/UserLogin.js [hunk 1]
```diff
@@ -40,35 +40,22 @@ const UserLogin = () => {
...
-        Don't have an account?{' '}
-        <span>
-          <button onClick={handleChange}>Sign up</button>
-        </span>
-      </p>
-      <LogInContainer logIn={logIn} handleSubmit={handleSubmit} />
-    </div>
+    <>
+      <LogInContainer
+        logIn={logIn}
```
Comment (ezerssss) [inline client/src/pages/UserLogin/UserLogin.js:54]: mas nice guro nga ang variable name is `isLogginIn` para mas explicit siya

File: client/src/pages/UserLogin/components/LogInContainer.js [hunk 2]
```diff
@@ -1,20 +1,112 @@
...
+  buttonTextSize,
+} from '../../../styles/userLoginStyles.js';
 
 const LogInContainer = (props) => {
-  const { logIn, handleSubmit } = props;
-  const text = logIn ? 'Log In' : 'Sign Up';
+  const { logIn, handleChange, handleSubmit } = props;
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const text = logIn ? 'LOG IN' : 'SIGN UP';
```
Comment (ezerssss) [inline client/src/pages/UserLogin/components/LogInContainer.js:16]: rename to a more descriptive variable

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: client/src/pages/UserLogin/UserLogin.js (hunk 1 above)
Comment (ezerssss) [reply client/src/pages/UserLogin/UserLogin.js:54]: if boolean ang variable usually naa jud na nga mga prefix, `is` `has` etc.

Comment (maxellmilay) [reply client/src/pages/UserLogin/UserLogin.js:54]: sige sige

**Review by ezerssss — APPROVED** [review]

**Pull request merged**